CONTRIBUTING.md

```diff
@@ -45,37 +37,73 @@ Please be sure to follow the usual process for submitting PRs:
 
 We reserve the right to close PRs that are not making progress. Closed PRs can be reopened again later and work can resume.
 
-### Contributor License Agreement
-By contributing your code to GNU Radio you grant us a non-exclusive,
-irrevocable, worldwide, royalty-free, sublicenseable, transferable
-license under all of Your relevant intellectual property rights
-(including copyright, patent, and any other rights), to use, copy,
-prepare derivative works of, distribute and publicly perform and
-display the Contributions on any licensing terms, including without limitation:
-(a) open source licenses like the Apache license; and (b) binary,
-proprietary, or commercial licenses. Except for the licenses granted herein,
-You reserve all right, title, and interest in and to the Contribution.
-
-You confirm that you are able to grant us these rights. You represent
-that You are legally entitled to grant the above license. If Your employer
-has rights to intellectual property that You create, You represent that
-You have received permission to make the Contributions on behalf of that
-employer, or that Your employer has waived such rights for the Contributions.
-
-You represent that the Contributions are Your original works of
-authorship, and to Your knowledge, no other person claims, or
-has the right to claim, any right in any invention or patent
-related to the Contributions. You also represent that You are
-not legally obligated, whether by entering into an agreement
-or otherwise, in any way that conflicts with the terms of this license.
-
-We acknowledge that, except as explicitly described in this
-Agreement, any Contribution which you provide is on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
-EITHER EXPRESS OR IMPLIED, INCLUDING, WITHOUT LIMITATION,
-ANY WARRANTIES OR CONDITIONS OF TITLE, NON-INFRINGEMENT,
-MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
+### Copyright Assignment
+
+GNU Radio does not claim ownership of any contributions you make. All copyrights remain with the original author of the contribution. As the author, you are responsible for maintaining and marking your copyright in the appropriate locations.
+
+When modifying or adding new files, you must include a copyright header in each file you touch. The copyright header should contain the following information:
+
+```
+/*
+ * Copyright (C) [Year] [Name or Pseudonym of Author]
+ *
+ * SPDX-License-Identifier: [License of module or file, default LGPL-3.0 for core]
```

Comment:
On a similar note as above:

In my view, it's important to keep licensing and (c)lean, maintainable code as separate concerns. Including a mandatory copyright header in every file doesn’t necessarily add value but does increase maintenance overhead and refactoring complexity.

Moreover, such headers can create unnecessary ambiguity, leading to questions about whether changes or add-ons conflict with top-level licensing agreements and a lot of bikeshedding. This ambiguity can require further reviews from legal experts, increasing the burden on contributors and maintainers.

To simplify this process, we already have a [contributor list](https://github.com/fair-acc/gnuradio4#user-content-contributors), which acknowledges contributions equally, regardless of the size or nature of the improvement. In line with the principle of "Keep It Simple, Stupid" (KISS), I believe we avoid legalese in the source code and focus on the core utility and quality of the library itself.

Reply:
I get your point, but GNU Radio 3.10 does this and it doesn't seem to generate much burden.